css/main.css

```diff
@@ -473,7 +473,8 @@ img {
     margin: 0;
     font-size: 18px;
     line-height: 24px;
-    padding: 22px 46px;
+    width: 200px;
+    height: 60px;
```

Comment:
Why do you need it? Can't we use paddings?

Reply:
I've changed it in all buttons